### ISSUE TITLE
Accessibility: Nav tag should have aria-label

### ISSUE THREAD

**Issue by lyubomir-popov** [canonical-web-and-design/vanilla-framework#3252]
- Nav tag should have aria-label (as in this [example](https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html))
